Assets/_BForBoss/_Analytics/Scripts/BForBossAnalytics.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections;
+
+namespace Perigon.Analytics
+{
+    #region STRUCTS
+    public readonly struct Event
+    {
+        public const String PlayerDeath = "Player - Death";
+        public const String CheckpointReached = "Checkpoint - Reached";
+    }
+
+    public readonly struct EventAttribute
+    {
+        public const String Course = "course";
+        public const String Name = "name";
+        public const String Time = "timer";
+    }
+    public readonly struct EventConstant
+    {
+        public const String RaceCourse = "racecourse";
+    }
+    public readonly struct Profile
+    {
+        public const string Name = "$name";
+        public readonly struct Controls
+        {
+            public const string MouseHorizontalSens = "mouse_horizontal_sens";
+            public const string MouseVerticalSens = "mouse_vertical_sens";
+            public const string ControllerHorizontalSens = "controller_horizontal_sens";
+            public const string ControllerVerticalSens = "controller_vertical_sens";
+            public const string Inverted = "inverted";
+        }
+    }
+    #endregion
+   
+    public interface IBForBossAnalytics
+    {
+        void StartSession(string uniqueId);
```

Comment:
NiT - u don't have to name this uniqueId 

Putting down Id implies it's supposed to be unique